dashboard/app/models/school_info.rb

```diff
@@ -151,7 +151,7 @@ def should_check_for_full_validation?
 
   def validate_zip
     if zip
-      errors.add(:zip, 'Invalid zip code') unless zip > 0
+      errors.add(:zip, 'Invalid zip code') unless zip > 0 && zip < 99999
```

Comment:
I believe this should be either
```suggestion
      errors.add(:zip, 'Invalid zip code') unless zip > 0 && zip <= 99999
```

or

```suggestion
      errors.add(:zip, 'Invalid zip code') unless zip > 0 && zip < 100_000
```

Reply:
committed the second change!